air/src/execution_step/air/mod.rs

```diff
@@ -141,11 +141,11 @@ macro_rules! log_instruction {
 /// This macro converts joinable errors to Ok and sets subgraph complete to false.
 #[macro_export]
 macro_rules! joinable {
-    ($cmd:expr, $exec_ctx:expr) => {
+    ($cmd:expr, $exec_ctx:expr, $ok_result:expr) => {
         match $cmd {
             Err(e) if e.is_joinable() => {
                 $exec_ctx.subgraph_complete = false;
-                return Ok(());
+                return Ok($ok_result);
```

Comment:
It seems all usages of `joinable!` still have `()` as last argument.  It might be useful anyway, but if I have overlooked something, or is it just remnants of some intermediate change?

Reply:
It's just silent "bug" fixing